gradle/libs.versions.toml

```diff
@@ -1,89 +1,90 @@
 [versions]
-assetModel = "0.1.2"
-assetSpec = "1.0.0"
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
```

Comment:
A few "real" changes here, but mostly this renames the dependencies to a more better format.  The effects can be seen in all the affected `build.gradle.kts` files across the application.  It's mostly an aesthetic thing.